deployment/ccip/changeset/cs_add_lane.go

```diff
@@ -43,6 +42,7 @@ type LaneConfig struct {
 	DestSelector          uint64
 	InitialPricesBySource InitialPrices
 	FeeQuoterDestChain    fee_quoter.FeeQuoterDestChainConfig
+	TestRouter            bool
```

Comment:
Just more flexible this way cc @AnieeG

Reply:
Wondering if we should allow this. If someone does not mention this TestRouter it will just add it to real router without any initial testing. In this way we will never allow adding a lane first to real router and rather will create sep proposal to enable real router.

Reply:
spoke offline - we can add defensive checks elsewhere